index.html

```diff
@@ -2412,7 +2412,17 @@ <h2>Versioning</h2>
 			<p class="issue" data-number="12">See corresponding open issue.</p>
 
 	</section>
-	
+
+	<section id="vdr-forks">
+		<h2>VDR Network Forks</h2>
+		<p>DID Methods that utilize VDRs of distributed systems (such as distributed ledgers or other distributed systems) need to manage the potential for network forks taking place. A <a>DID method</a> that uses a distributed system as a <a>verifiable data registry</a> SHOULD disambiguate the VDR they are using in the case of such forks by either:</p>
```

Comment:
```suggestion
		<p><a>DID methods</a> that utilize VDRs of distributed systems (such as distributed ledgers or other distributed systems) need to manage the potential for network forks taking place. A <a>DID method</a> that uses a distributed system as a <a>verifiable data registry</a> SHOULD disambiguate the VDR they are using in the case of such forks by either:</p>
```

Reply:
I wonder if this SHOULD statement is better here - https://www.w3.org/TR/did/upcoming/#methods
Although, since we can only do class 3 changes perhaps that is not possible in this iteration.